student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, classroom, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] The `parent_permission` parameter uses the colon notation so we should update the assignment in the super method correctly.

  ```
  super(age, name, parent_permission: parent_permission)
  ```